packages/core/admin/ee/server/controllers/admin.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
       currentActiveUserCount,
       permittedSeats,
       shouldNotify,
-      shouldStopCreate: currentActiveUserCount >= permittedSeats,
+      shouldStopCreate: permittedSeats && currentActiveUserCount >= permittedSeats,
```

Comment:
What happens if permiettedSeats = 0? Do we end up with the wanted behavior?

Reply:
I'll do a stricter check but in practice it's either > 0 or null|undefined